profiles/common/modules/features/multisite_notifications/multisite_notifications_core/multisite_notifications_core.module

```diff
@@ -38,8 +32,11 @@ function multisite_notifications_core_subscriptions_queue_alter(&$event) {
  * Implements hook_workbench_moderation_transition().
  */
 function multisite_notifications_core_workbench_moderation_transition($node, $previous_state, $new_state) {
+  // Only process events after publishing
+  // See multisite_notifications_core_subscriptions_queue_alter()
   if ($new_state == workbench_moderation_state_published()) {
     $full_node = node_load($node->nid);
+    $full_node->flag_after_publishing = TRUE;
```

Comment:
As stated before, I don't see an advantage, but you can do it if you think it improves, no problem.